.github/workflows/torchao.yml

```diff
@@ -0,0 +1,86 @@
+name: torchao
+
+on:
+  push:
+    tags:
+      - ciflow/torchao/*
+  workflow_dispatch:
+    inputs:
+      noquant:
+        description: Run noquant?
+        required: false
+        type: boolean
+        default: true
+      int8dynamic:
+        description: Run int8dynamic?
+        required: false
+        type: boolean
+        default: true
+      int8weightonly:
+        description: Run int8weightonly?
+        required: false
+        type: boolean
+        default: true
+      int4weightonly:
+        description: Run int4weightonly?
+        required: false
+        type: boolean
+        default: true
+      autoquant:
+        description: Run autoquant?
+        required: false
+        type: boolean
+        default: true
+      benchmark_configs:
+        description: The list of configs used the benchmark
+        required: false
+        type: string
+        default: torchao_huggingface_perf,torchao_timm_perf,torchao_torchbench_perf
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref_name }}-${{ github.ref_type == 'branch' && github.sha }}-${{ github.event_name == 'workflow_dispatch' }}-${{ github.event_name == 'schedule' }}
+  cancel-in-progress: true
+
+permissions: read-all
+
+jobs:
+  linux-focal-cuda12_1-py3_10-gcc9-torchao-build:
+    name: cuda12.1-py3.10-gcc9-sm80
+    uses: ./.github/workflows/_linux-build.yml
+    with:
+      build-environment: linux-focal-cuda12.1-py3.10-gcc9-sm80
+      docker-image-name: pytorch-linux-focal-cuda12.1-cudnn8-py3-gcc9-inductor-benchmarks
+      cuda-arch-list: '8.0'
+      test-matrix: |
+        { include: [
+          { config: "torchao_huggingface_perf", shard: 1, num_shards: 3, runner: "linux.gcp.a100.large" },
```

Comment:
I assume that this will x2 the traffic to our A100 fleet, so I don't have full confidence about the queueing situation.  But then, let's see how bad it is and goes from there if there is no strong objection.  Maybe we want to reduce the number of models to cover here instead of doing a full rollout.

Reply:
Currently we are planning to only enable it upon manual trigger or PR with tag `ciflow/torchao`. If nightly run is a problem for the fleet, we could make it every 3 days or every week.